src/VisualStudio.Roslyn.SDK/ComponentDebugger/ProjectUtilities.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem;
+
+namespace Roslyn.ComponentDebugger
+{
+    public static class ProjectUtilities
+    {
+        // PROTOTYPE: is there a way to get this other than hardcoding it?
+        static readonly string[] CommandLineSchemaRuleNames = new[] { "CompilerCommandLineArgs" };
+
+        public static async Task<IList<string>> GetCompilationArgumentsAsync(this ConfiguredProject project)
```

Comment:
📝This is a place I'd like to get the project system folks to take a look at how we're getting the value.